lib/go-atscfg/remapdotconfig_test.go

```diff
@@ -585,209 +578,6 @@ func TestMakeRemapDotConfigDuplicateOrigins(t *testing.T) {
 	}
 }
 
-func TestMakeRemapDotConfigNilMidRewrite(t *testing.T) {
```

Comment:
Shouldn't this test still occur, just without setting/checking a CacheURL value?

Reply:
I originally had removed this test because the Nil config returns nothing (ignoring comments), I've readded it and just asserted that only the comment is present